### ISSUE TITLE
North Atlantic Ocean

### ISSUE THREAD

**Issue by missinglink** [whosonfirst-data/whosonfirst-data#1084]
Heya,

The geometry of the [North Atlantic Ocean](https://spelunker.whosonfirst.org/id/404528709/) (404528709) flows into part of Manhattan:

![ss](https://user-images.githubusercontent.com/738069/37100278-cfac9a10-2222-11e8-9ae8-def61e62ddf2.png)

Is there anything we can do about this? It messes up some of my PIP results :)

**Comment by stepps00** [comment]
Yeah... most of the `ocean` and `marinearea` polygons have less-precise geometries that overlap land. A fix could be to either find new geometries for those placetypes or clip them to other admin boundaries to remove overlap.

Whatever we decide, we should retain the existing geometries as alt-geometries if we decide to update them.
